customize.dist/translations/messages.es.js

```diff
@@ -337,25 +337,25 @@ define(function () {
     out.printDate = "Mostrar la fecha";
     out.printTitle = "Mostrar el título";
     out.printCSS = "CSS personalizado:";
-    out.editOpen = "Abrir enlances de edición en pestaña nueva";
+    out.editOpen = "Abrir enlaces de edición en pestaña nueva";
     out.editOpenTitle = "Abrir en modo edición en pestaña nueva";
     out.settings_importTitle = "Importar pads recientes locales en CryptDrive";
     out.settings_import = "Importar";
-    out.settings_importConfirm = "¿Seguro qué quieres importar tus pads recientes a tu cuenta CryptDrive?";
+    out.settings_importConfirm = "¿Seguro que quieres importar tus pads recientes a tu cuenta CryptDrive?";
     out.settings_importDone = "Importación terminada";
 
     out.tips = {};
     out.tips.lag = "El icono verde en la parte superior derecha muestra la calidad de tu connexión a CryptPad.";
     out.tips.shortcuts = "`ctrl+b`, `ctrl+i`, y `ctrl+u` son accesos rápidos para negrita, itálica y subrayado.";
-    out.tips.indent = "Cuando editas listas, puedes usar tab o shift+tab para icrementar o decrementar indentación.";
+    out.tips.indent = "Cuando editas listas, puedes usar tab o shift+tab para incrementar o decrementar la sangría.";
```

Comment:
I trusted my Microsoft Word and LibreOffice Writer installations, which have both translated "identation" as "sangría", apparently in the same context as in CryptPad. To be honest, I've never heard the term "diagramation design". On the other hand, "margen" has always sounded more natural to me. So I don't know what to say.